packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSocialLink.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from '../LoginPageComponents/LoginPageLink';
+import { noop } from '../../../../common/helpers';
```

Comment:
travis caught this line, just a bit of clean up

Reply:
Thanks, fixed :)